app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -32,22 +32,23 @@ class BrowserRobot {
             .check(matches(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
     }
 
-    fun verifyHelpUrl() {
+    fun verifyUrl(redirectUrl: String) {
         val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
-        val redirectUrl = "https://support.mozilla.org/"
-        mDevice.waitNotNull(Until.findObject(By.res("org.mozilla.fenix.debug:id/mozac_browser_toolbar_url_view")),
-            TestAssetHelper.waitingTime)
+        mDevice.waitNotNull(Until.findObject(By.res("org.mozilla.fenix.debug:id/mozac_browser_toolbar_url_view")), TestAssetHelper.waitingTime)
         onView(withId(R.id.mozac_browser_toolbar_url_view))
             .check(matches(withText(containsString(redirectUrl))))
     }
 
+    fun verifyHelpUrl() {
+        verifyUrl("https://support.mozilla.org/")
+    }
+
     fun verifyWhatsNewURL() {
-        val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
-        val redirectUrl = "https://support.mozilla.org/"
-        mDevice.waitNotNull(Until.findObject(By.res("org.mozilla.fenix.debug:id/mozac_browser_toolbar_url_view")),
-            TestAssetHelper.waitingTime)
-        onView(withId(R.id.mozac_browser_toolbar_url_view))
-            .check(matches(withText(containsString(redirectUrl))))
+        verifyUrl("https://support.mozilla.org/")
+    }
+
+    fun verifyRateOnGooglePlayURL() {
+        verifyUrl("https://play.google.com/store/")
```

Comment:
When is this used?

Reply:
This is used when the Google Play app is not installed on the device. It will try to open the url in the browser. 

The app id is missing in the url though so I will update.